backend/tracim_backend/lib/search/elasticsearch_search/elasticsearch_search.py

```diff
@@ -482,20 +603,42 @@ def _create_ingest_pipeline(self) -> None:
         # TODO - G.M - 2019-05-31 - check if possible to set specific analyzer for
```

Comment:
You can remove this TODO, but may be interesting to think about doing lang specific indexing for `raw_content` (and `description` ?) field too.
i suggest to create a new TODO (and a specific issue) about lang specific indexing for thoses. The difficulty is how we guess the lang, for file guess, we do use data given by ingest (using apache tikka), we do not have direct access to apache tikka outside so, we may need a new guesser for it.
Note that if we did such guesser, it may be useful for improving 
For now, as we do not have good guesser mecanism, it's better to rely on default analyzer for `raw_content` and `description`

Reply:
https://github.com/tracim/tracim/issues/4184